js/apps/admin-ui/src/realm-settings/security-defences/BruteForceDetection.tsx

```diff
@@ -152,6 +156,14 @@ export const BruteForceDetection = ({
               bruteForceMode ===
                 BruteForceMode.PermanentAfterTemporaryLockout) && (
               <>
+                <SelectControl
+                  name="bruteForceStrategy"
+                  label={t("bruteForceStrategy")}
+                  labelIcon={t("bruteForceStrategyHelp")}
+                  controller={{ defaultValue="MULTIPLE" }}
```

Comment:
Pretty sure this is not valid JavaScript. Let's also re-use `bruteForceStrategyTypes` here.
```suggestion
                  controller={{ defaultValue: bruteForceStrategyTypes[0] }}
```